src/main/resources/db/migration/V1_72__uniqu_gjelden_oppfolging.sql

```diff
@@ -0,0 +1,4 @@
+delete from OPPFOLGINGSPERIODE a1 where exists(select * from  OPPFOLGINGSPERIODE a2 where a1.AKTOR_ID = a2.AKTOR_ID and  a1.SLUTTDATO = a2.SLUTTDATO  and  a1.STARTDATO < a2.STARTDATO);
+delete from OPPFOLGINGSPERIODE a1 where exists(select * from  OPPFOLGINGSPERIODE a2 where a1.AKTOR_ID = a2.AKTOR_ID and  a1.SLUTTDATO is null  and a2.SLUTTDATO is null  and  a1.STARTDATO < a2.STARTDATO);
```

Comment:
Hadde det vært bedre å kjøre dette som et script ved siden av? F.eks med hjelp av DBA

Reply:
Jeg lurer på om det holder å kjøre 
`delete from OPPFOLGINGSPERIODE a1 where exists(select * from OPPFOLGINGSPERIODE a2 where a1.AKTOR_ID = a2.AKTOR_ID and a1.SLUTTDATO = a2.SLUTTDATO and a1.STARTDATO < a2.STARTDATO);`
Siden den vil også sjekke at begge er null.

Reply:
null = null == false I sql/oracle